books/architecture/src/appendix/crates.md

```diff
@@ -0,0 +1,61 @@
+# Crates
+This is an index of all of Cuprate's in-house crates it uses and maintains.
+
+They are categorized into groups.
+
+Crate documentation for each crate can be found by clicking the crate name or by visiting <https://doc.cuprate.org>. Documentation can also be built manually by running this at the root of the `cuprate` repository:
+```bash
+cargo doc --package $CRATE
+```
+For example, this will generate and open `cuprate-blockchain` documentation:
+```bash
+cargo doc --open --package cuprate-blockchain
+```
+
+## Consensus
+| Crate | In-tree path | Purpose |
+|-------|--------------|---------|
+| [`cuprate-consensus`](https://doc.cuprate.org/cuprate_consensus) | [`consensus/`](https://github.com/Cuprate/cuprate/tree/main/consensus) | TODO
```

Comment:
There are some `TODO`s for crate purposes. We can leave these for now or fill them, although I'm not sure what a good 1-line description would be for some of these.

Reply:
I'll fill them in in a future PR